update readme (#5)
DevExpress-Examples/xaf-how-to-use-data-from-security-in-criterion: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3945)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# XAF - How to use data from a securiy user's property in a criterion

> This example shows how to use data from the currently logged-in user in a criterion (e.g. in a validation rule).

Reviewer: ```suggestion
This example shows how to use data from the currently logged-in user in a criterion, such as a validation rule.
```

![image](https://github.com/DevExpress-Examples/XAF_how-to-create-a-custom-function-criteria-operator-e3945/assets/14300209/979f25d7-23d3-4f60-a30f-18e61f8e0d7b)

## Implementation Details

> As there is no way to access security system data from a criterion, you should patch a criterion using the `DevExpress.ExpressApp.Security.SecurityEvents.OnCustomizeSecurityCriteriaOperator` event. 

Reviewer: ```suggestion
Since criterion security system data is not accessible, you should patch a criterion using the `DevExpress.ExpressApp.Security.SecurityEvents.OnCustomizeSecurityCriteriaOperator` event. 
```
